presto-native-execution/etc/config.properties

```diff
@@ -2,4 +2,5 @@ discovery.uri=http://127.0.0.1:58215
 presto.version=testversion
 http-server.http.port=7777
 shutdown-onset-sec=1
-register-test-functions=true
\ No newline at end of file
+register-test-functions=true
+shuffle.name=local
```

Comment:
Unrelated change?

Reply:
Related actually.
We need shuffle.name set to a default value for the unit test to run. shuffle.name set as local seems like the best default value.

Reply:
No, this is not needed for unit tests to run. Please, remove this change.

Reply:
@mbasmanova when running in detached mode, this property is indeed needed. Otherwise native cpp runs without any shuffle client.

Reply:
Sure, but this should be a separate PR and it requires splitting Prestissimo and Presto-on-Spark servers into separate binaries with separate set of configs.